src/test/java/Helloworld/HelloWorldTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class HelloWorldTest
+{
+	private static final String s="java";
+	HelloWorld h=new HelloWorld();
+	@Test
+	public void testsayHelloWhenInputParamIsString()
+	{
+		assertEquals("hello world java",h.sayHello(s));
+	}
+	//passing null string
+	@Test(expected=IllegalArgumentException.class)
+	public void testsayHelloWhenInputParamIsNull()
+	{
+		h.sayHello(null);
```

Comment:
This is good. We can make it better by adding `fail("should throw exception");` after h.sayHello(..) so that anyone looking at the code know that this method asserts by expecting exeption.